Minio.Examples/Cases/PutObject.cs

```diff
@@ -38,27 +38,28 @@ class PutObject
                 byte[] bs = File.ReadAllBytes(fileName);
                 using (MemoryStream filestream = new MemoryStream(bs))
                 {
-                    if (filestream.Length < (5 * MB))
+                    FileInfo fileInfo = new FileInfo(fileName);
+                    if (fileInfo.Length < (5 * MB))
                     {
                         Console.WriteLine("Running example for API: PutObjectAsync with Stream");
                     }
                     else
                     {
                         Console.WriteLine("Running example for API: PutObjectAsync with Stream and MultiPartUpload");
```

Comment:
multipart is not even happening here - remove this else

Reply:
Yes. Removed.